app/controllers/access_requests_controller.rb

```diff
@@ -40,13 +40,17 @@ def preview
 
   def create
     @emailed_access_request = EmailedAccessRequest.new(emailed_access_request_params)
-
     begin
-      @emailed_access_request.manually_approve!
+      access_request_params = emailed_access_request_params.to_h
+      access_request_params[:email_address] = access_request_params.delete(:target_email)
+      access_request = AccessRequestAPI.new(access_request_params)
+      access_request.save
+      access_request.approve
```

Comment:
What if the `.save` or `.approve` here fail?

Reply:
We test 401, 404, 999 separately